toolbox/githubctl/main.go

```diff
@@ -252,7 +253,7 @@ func ReleasePipelineRelease(baseBranch *string) error {
 		_, err := u.Shell(copyCmd)
 		return err
 	}
-	_, err := ghClntRel.CreatePRUpdateRepo(srcBranch, dstBranch, dailyRepo, prTitle, prBody, edit)
+	_, err := ghClntRel.CreatePRUpdateRepo(srcBranch, dstBranch, pipelineRepo, prTitle, prBody, edit)
```

Comment:
are these going to break the existing pipeline?

Reply:
no, this is not live, used only for the new system

Reply:
Looking at these few new methods, I think you should probably create a new package for release tooling, instead of overloading githubctl.

Can be done in a follow up PR

Reply:
will do